index.js

```diff
@@ -5,12 +5,14 @@ var React = require('react');
 var EXT_REGEX = new RegExp('\\.jsx$');
 var DEFAULTS = {
     doctype: '<!DOCTYPE html>',
-    renderMethod: 'renderToStaticMarkup',
-    layoutMethod: 'renderToStaticMarkup',
+    viewRenderMethod: 'renderToStaticMarkup',
```

Comment:
@jedireza this renaming of `renderMethod` to `viewRenderMethod` will cause a backwards incompatibility, which otherwise this PR does not cause.